README.md

```diff
@@ -0,0 +1,69 @@
+# Witaj na stronie Microsoft Azure User Group Poland
+
+Jesteśmy otwarci na naszą społeczność i dlatego też oddajemy stronę w Wasze ręcę. Każdy może uczestniczyć w tworzeniu tej strony poprzez naprawienie błędów (jeżeli jakieś pomineliśmy), ale przede wszystkim przez publikacje na tej stronie. Dlatego jeżeli masz pomysł na artykuł i chcesz go zamieścić na stronie grupy to zapraszam się do [poradnika kontrybucji](#Poradnik-kontrybucji). Pamiętaj, że grupa zrzesza nie tyle co ekspertów Azure, co **miłośników** tej chmury. Każdy kiedyś zaczynał i każdy stawiał pierwsze kroki. Będziemy bardzo zadowloeni, jeżeli to właśnie z nami zaczniesz swoją przygodę!
+
+
+## Poradnik kontrybucji
```

Comment:
Dodaj jeszcze info o dodaniu się do https://github.com/AzureCommunityPL/azurecommunitypl.github.io/blob/master/_data/authors.yml

Reply:
To jest niżej

```
6. Jeżeli jest to twoja pierwsza kontrybucja dodaj parę słów o sobie w pliku `_data/authors.yml`
```

Reply:
Chodziło mi o brak przykładu co tam wpisać.